daphne_worker/src/router/leader.rs

```diff
@@ -172,7 +172,9 @@ pub(super) fn add_leader_routes(router: DapRouter<'_>) -> DapRouter<'_> {
                     Ok(DapCollectJob::Done(collect_resp)) => dap_response_to_worker(DapResponse {
                         version: req.version,
                         media_type: DapMediaType::Collection,
-                        payload: collect_resp.get_encoded_with_param(&req.version),
+                        payload: collect_resp
+                            .get_encoded_with_param(&req.version)
+                            .expect("XXX"),
```

Comment:
@mendess how should we handle this encoding failure? Previously we would have panicked.

Reply:
same as other routing method